app/models/video.py

```diff
@@ -0,0 +1,26 @@
+from flask import current_app
+from flask.helpers import make_response
+from app import db
+from sqlalchemy import DateTime
+
+class Video(db.Model):
+    __tablename__ = 'videos'
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime, nullable=True)
+    total_inventory = db.Column(db.Integer)
+    # available_inventory = db.Column(db.Integer,default=0, nullable = True)
+    # returns a list of customers linked to this particular video (since multiple copies)
+    customers = db.relationship('Customer', back_populates='videos', secondary='rentals')
+    # rentals = db.relationship('Rental', backref='rental', lazy=True)
```

Comment:
Again, let's swap these out.

```suggestion
    rentals = db.relationship('Rental', backref='rental', lazy=True)
```